src/tests/test_typeconversions.py

```diff
@@ -64,6 +66,46 @@ def test_string_conversion2(selenium, s):
     assert run_js("""(spy) => spy === self.sjs""")(s)
 
 
+def blns():
+    yield ""
+    import base64
+    import json
+    from urllib.request import urlopen
+
+    try:
+        with urlopen(
+            "https://raw.githubusercontent.com/minimaxir/big-list-of-naughty-strings/master/blns.base64.json"
```

Comment:
The list of strings are shorter than I think, so perhaps we can embed it instead of downloading it?

Reply:
+1 possibly under `pyodide/src/tests/` but maybe gzip-compressed, to avoid appearing in grep.